packages/next/src/generators/next-auth-redis/schema.json

```diff
@@ -0,0 +1,35 @@
+{
+    "$schema": "http://json-schema.org/schema",
+    "cli": "nx",
+    "$id": "NextAuth",
+    "title": "Ensono Stacks Next Auth Redis Generator",
+    "description": "The @ensono-stacks/next:next-auth-redis generator will add Redis for session management in a Next-Auth app.",
+    "type": "object",
+    "properties": {
+        "project": {
+            "type": "string",
+            "description": "The name of the project.",
+            "alias": "p",
+            "$default": {
+                "$source": "projectName"
+            },
+            "x-prompt": "What is the name of the project?",
+            "x-priority": "important"
+        },
+        "envVar": {
+            "type": "string",
+            "default": "REDIS_URL",
+            "description": "Name of the env var that stores connection string for Redis",
+            "x-prompt": "What is the name of the env var that stores connection string for Redis?",
+            "pattern": "^[A-Z_]+$"
```

Comment:
Regex pattern accepts `_ABC`. Could this be changed to` ^[A-Z]+(?:_[A-Z]+)*$` so it only accepts `ABC_DEF`, `ABC`